concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1237,6 +1250,38 @@ jobs:
                 export CONCOURSE_URL="https://${CONCOURSE_HOSTNAME}.${SYSTEM_DNS_ZONE_NAME}"
                 export CONCOURSE_ATC_PASSWORD
                 CONCOURSE_ATC_PASSWORD=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.concourse_atc_password concourse-secrets/concourse-secrets.yml)
+                export GITHUB_CLIENT_ID
+                export GITHUB_CLIENT_SECRET
+
+                if [ "${ENABLE_GITHUB}" = "true" ] ; then
+                  GITHUB_CI_USERS=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb ci ./paas-bootstrap/concourse/users/users.yml)
+                  GITHUB_PROD_USERS=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb prod ./paas-bootstrap/concourse/users/users.yml)
+                  GITHUB_CLIENT_ID=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.github_client_id github-oauth-secrets/github-oauth-secrets.yml)
+                  GITHUB_CLIENT_SECRET=$(./paas-bootstrap/concourse/scripts/val_from_yaml.rb secrets.github_client_secret github-oauth-secrets/github-oauth-secrets.yml)
+
+                  if [ "${FLY_TEAM}" = "dev" ] || [ "${FLY_TEAM}" = "ci" ]; then
+                    export GITHUB_USERS="${GITHUB_CI_USERS} ${GITHUB_PROD_USERS}"
+                  else
+                    export GITHUB_USERS="${GITHUB_PROD_USERS}"
+                  fi
+
+                  ./paas-bootstrap/concourse/scripts/fly_sync_and_login.sh
+                  # shellcheck disable=SC2046,SC2116
+                  ${FLY_CMD} -t "${FLY_TARGET}" set-team -n "${FLY_TEAM}" \
+                    --github-auth-client-id "${GITHUB_CLIENT_ID}" \
+                    --github-auth-client-secret "${GITHUB_CLIENT_SECRET}"  \
+                    $(echo "$GITHUB_USERS") --basic-auth-username admin --basic-auth-password "${CONCOURSE_ATC_PASSWORD}" --non-interactive
```

Comment:
I don't think we need to set this up for this team (only for the main team). We never log into this team, it only exists to serve as in indicator for which environment you're logging into (to help avoid prod being confused with dev for example)

Reply:
Addressed in https://github.com/alphagov/paas-bootstrap/pull/102